### PR TITLE
fix(role-migration): revamp old migration process and propose to call support instead

### PR DESCRIPTION
**Pull request by SteveBunlon**

### CODE REVIEW

File: how-tos/maintain/migrate-to-the-new-role-system.md [hunk 1]
```diff
@@ -45,49 +45,14 @@ Please check your agent type and version and read on or switch to the right docu
...
+{% hint style="success" %}
+If you still have access to your project today, you are using the new role system already, read more about **Roles** in our [User Guide](https://docs.forestadmin.com/user-guide/project-settings/teams-and-users/manage-roles).
 {% endhint %}
 
-{% hint style="info" %}
-You can read more about **Roles** in our [User Guide](https://docs.forestadmin.com/user-guide/project-settings/teams-and-users/manage-roles).
+{% hint style="danger" %}
+The old role system has been deprecated the 1st of June 2023, has reached its end of life the 1st of December 2023, and support has been dropped entirely the 4th of December 2024. Please do note that the new Role permissions system requires that you use **version 6.6+** of your agent (version **5.4+** for Rails) on **all** your environments. If you are running proper versions and urgently need to migrate to the new Roles system please contact our [support](mailto:support@forestadmin.com).
```
Comment (DayTF) [inline how-tos/maintain/migrate-to-the-new-role-system.md:53]:
```suggestion
The old role system has been deprecated the 1st of June 2023, has reached its end of life the 1st of December 2023, and support has been dropped entirely the 4th of December 2024. Please do note that the new Role permissions system requires that you use **version 6.6+** of your agent (**version 5.4+** for Rails) on **all** your environments. If you are running proper versions and urgently need to migrate to the new Roles system please contact our [support](mailto:support@forestadmin.com).
```

File: how-tos/maintain/migrate-to-the-new-role-system.md [hunk 2]
```diff
@@ -45,49 +45,14 @@ Please check your agent type and version and read on or switch to the right docu
 {% endtabs %}
 
 # Migrate to the new role system
-
```
Comment (DayTF) [inline how-tos/maintain/migrate-to-the-new-role-system.md:48]: Remove title right on top of this line ? it is redundant with the page title?

File: how-tos/maintain/migrate-to-the-new-role-system.md (hunk 1 above)
Comment (DayTF) [reply how-tos/maintain/migrate-to-the-new-role-system.md:53]: to be consistent with **version 6.6+**

File: how-tos/maintain/migrate-to-the-new-role-system.md (hunk 2 above)
Comment (SteveBunlon) [reply how-tos/maintain/migrate-to-the-new-role-system.md:48]: @arnaudbesnier is the owner of this, let's leave this as it is and change the format for all pages in another PR

**Review by DayTF — APPROVED** [review]

**Pull request merged**